conformance/conformance_test.go

```diff
@@ -52,6 +52,9 @@ func TestConformance(t *testing.T) {
 		GatewayClassName: *gatewayClassName,
 		Debug:            *showDebug,
 		Cleanup:          *shouldCleanup,
+		ExtendedSupport: []suite.ConformanceTestSupport{
+			suite.SupportReferencePolicy,
+		},
```

Comment:
Nit: I think we should just call this "Features" or something more generic, and not opt-in to any extended feature testing by default.

```suggestion
		Features: []suite.ConformanceTestFeatures{},
```